GitDepend/CommandLine/BuildSubOptions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using CommandLine;
+
+namespace GitDepend.CommandLine
+{
+	/// <summary>
+	/// The options for the build verb.
+	/// </summary>
+	public class BuildSubOptions : CommonSubOptions
```

Comment:
We talked about this, the Named Dependency visitor does not visit the base project, so it doesn't work well for this verb.